indexing-service/src/main/java/io/druid/indexing/common/task/HadoopTask.java

```diff
@@ -84,27 +107,35 @@ protected ClassLoader buildClassLoader(final TaskToolbox toolbox) throws Excepti
     final List<URL> nonHadoopURLs = Lists.newArrayList();
     nonHadoopURLs.addAll(Arrays.asList(((URLClassLoader) HadoopIndexTask.class.getClassLoader()).getURLs()));
 
-    final List<URL> driverURLs = Lists.newArrayList();
-    driverURLs.addAll(nonHadoopURLs);
+    final List<URL> druidURLs = ImmutableList.copyOf(
```

Comment:
jars added to the Job are nonHadoopURLs

The nonHadoopURLs are split into druidURLs and nonDruidURLs which are only used in the ClassLoader for launching the job via runTask.